dev/pastefromwordimage/getclipboard.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Get Clipboard HTML and RTF</title>
+    <style>
+
+        textarea {
+            border: 1px solid #808080;
+            float: left;
+            height: 200px;
+            width: 100%;
+            overflow: auto;
+            margin-bottom: 20px;;
+        }
+
+    </style>
+	<script src="../../ckeditor.js"></script>
+</head>
+<body>
+    <p>Paste Inside the Editor:</p>
+    <div><textarea id="input"></textarea></div>
+	<div>
+		<div style="float: left; width: 49%">
+			<p>Raw HTML Data Received:</p>
+			<textarea data-name="input.html" id="rawHtml" readonly="readonly"></textarea>
+			<button id="htmlData">Save HTML Data</button>
+		</div>
+		<div style="float: right; width: 49%">
+			<p>Raw RTF Data Received:</p>
+			<textarea data-name="input.rtf" id="rawRtf" readonly="readonly"></textarea>
+			<button id="rtfData">Save RTF Data</button>
+		</div>
+	</div>
+	<div style="width: 100%; float: left;">
+		<p>After Paste Processing:</p>
+		<textarea id="output" readonly="readonly"></textarea>
+	</div>
```

Comment:
Maybe you could also add a container which will show all images pasted to the CKEditor (in the form already transformed by CKEditor), something like `Extracted Images`? If it's relatively easy to do (like 0,5h) you could take a look at it. From the other hand you can see pasted images inside editor instance, so not sure if it is needed. WDYT?

Reply:
On this stage of implementation it might be problematic. I could process RTF clipboard, but it might contains also Word Shapes, which are ignored for processing while pasting. That's why preparing similar solution here might be repeating functionality of the plugin, which seems to be not a good thing.
It should be easier, when further changes related to file transfer will be implemented:
https://github.com/ckeditor/ckeditor-dev/blob/128516a52485505176189743962eaf97d52aa067/plugins/pastefromwordimage/plugin.js#L60-L63
There is nice event when will be exposed processed URL and image which is going to be embed. With that implementation it should like 0,5h of work :)

Reply:
@msamsel Ok, so let's leave it as it is for now.